src/drivers/imx/sdma.c

```diff
@@ -437,6 +437,7 @@ static struct dma_chan_data *sdma_channel_get(struct dma *dma,
 		cdata->hw_event = -1;
 
 		channel->status = COMP_STATE_READY;
+		channel->index = i;
```

Comment:
Didn't I initialize the channel indices at driver probe time? Still, good to have, thanks.